### PR TITLE
Update chain.json

### PR DESCRIPTION
**Pull request by nochanc3**
Added MMS team endpoints and seed node

### CODE REVIEW

**Review by jlehtimaki — CHANGES_REQUESTED** [review]
resolve the conflicts

**Comment by nochanc3** [comment]
> resolve the conflicts

Resolved

File: archway-1/chain.json [hunk 1]
```diff
@@ -258,6 +271,10 @@
       {
         "address": "grpc-archway.mzonder.com:443",
         "provider": "MZONDER"
+      },
+      {
+        "address": "https://grpc-archway.mms.team:443",
```
Comment (jlehtimaki) [inline archway-1/chain.json:276]: Please remove the `https://` from the address since it's not used in the protocol

**Review by jlehtimaki — CHANGES_REQUESTED** [review]

**Comment by jlehtimaki** [comment]
I'm closing this since there's no activity

**Pull request closed without merging**